types/injectpromise/tslint.json

```
@@ -0,0 +1,15 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "npm-naming": [
+            true,
+            {
+                "mode": "code",
+                "errors": [
+                    ["NeedsExportEquals", false],
+                    ["NoDefaultExport", false]
+                ]
+            }
+        ]
+    }
```

```suggestion
    "extends": "@definitelytyped/dtslint/dt.json"
```
no longer needed when you correct module exports

---

Thank you for your review. I intentionally disable error checking because I want to import as default.

the way error checking suggest turns out using require()

```
import injectpromise = require('injectpromise')
```